src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -1206,27 +1538,30 @@ export class SavedObjectsRepository {
     objects: SavedObjectsBulkResolveObject[],
```

Comment:
Was this function intentionally excluded from `repository.security_extension.test.ts`?

Reply:
Nevermind, I found these tests elsewhere!